### DISCUSSION TITLE
What is simulation mode?

### DISCUSSION THREAD

**Discussion by GGlitchX** [CosmicHikari/Weather2-Remastered#67] [Questions] [answered]
I've heard of simulation mode and seen it in the config, what does it do?

**Comment by Mrbt0907** [comment, marked as answer]
In the current build, it breaks storms and all clouds turn into thuderstorms. I forgot to remove the config before I released the update.

In the next build, simulation mode will turn all storm systems into more realistic storms by using global variables like humidity and temperature. The storm progression will no longer be 100% randomized as they gain strength or lose strength based on various variables of the area.

For this build, I am not expecting it to match up close to real storms, but I want to take a step forward in that direction with simulation mode. I will need guidance from people who know how storms work once I begin finishing simulation mode.

**Reply by GGlitchX** [reply]
I can help with some aspects of storms , but I'm not sure if that's very viable to make it exactly like real life due to that just being very hard to code, although if you are up for the challenge I guess that would be cool. I'd kind of do it like what I explained in my frontal system / pressure suggestion with modifying chances based on variables per chunk. I completely forgot to mention temperature and humidity in my suggestion and a few other important things, so if you ever decide to do something similar to what I suggested then you might wanna add that it, like what you said in your reply. Once again, I'm not the developer, you are. You have full responsibility.
